run/filesystem/test/e2e-test.cloudbuild.yaml

```diff
@@ -0,0 +1,131 @@
+steps:
+- id: 'Set gcloud config values'
+  name: 'gcr.io/cloud-builders/gcloud:latest'
+  entrypoint: /bin/bash
+  args:
+  - '-c'
+  - |
+    gcloud config set run/region $_REGION && \
+    gcloud config set filestore/zone $_ZONE || \
+    echo 'Failed to set gcloud config values.' | tee -a /workspace/errors.txt
+
+- id: 'Create Filestore instance'
+  name: 'gcr.io/cloud-builders/gcloud:latest'
+  entrypoint: /bin/bash
+  waitFor: ['Set gcloud config values']
+  args:
+  - '-c'
+  - |
+    gcloud filestore instances create ${_INSTANCE_ID} \
+    --tier=STANDARD \
+    --file-share=name=${_SHARE_NAME},capacity=1TiB \
+    --network=name="default" || \
+    echo 'Failed to create Filestore instance.' | tee -a /workspace/errors.txt
+
+- id: 'Create serverless VPC connector'
+  name: 'gcr.io/cloud-builders/gcloud:latest'
+  entrypoint: /bin/bash
+  waitFor: ['-']
+  args:
+  - '-c'
+  - |
+    gcloud compute networks vpc-access connectors create \
+    ${_CONNECTOR_NAME} --region ${_REGION} --range "10.8.0.0/28" || \
+    echo 'Failed to create serverless VPC connector.' | tee -a /workspace/errors.txt
+
+- id: 'Retrieve Filestore IP address'
+  name: 'gcr.io/cloud-builders/gcloud:latest'
+  entrypoint: /bin/bash
+  waitFor: ['Create Filestore instance']
+  args:
+  - '-c'
+  - |
+    gcloud filestore instances \
+    describe $_INSTANCE_ID \
+    --format "value(networks.ipAddresses[0])" > /workspace/FILESTORE_IP_ADDRESS || \
+    echo 'Failed to retrieve Filestore IP address.' | tee -a /workspace/errors.txt
+
+- id: 'Build and deploy to Cloud Run'
+  name: 'gcr.io/cloud-builders/gcloud:latest'
+  entrypoint: /bin/bash
+  args:
+  - '-c'
+  - |
+    export FILESTORE_IP_ADDRESS=$(cat /workspace/FILESTORE_IP_ADDRESS) && \
+    gcloud run deploy $_RUN_SERVICE --source . \
+    --vpc-connector $_CONNECTOR_NAME \
+    --execution-environment gen2 \
+    --allow-unauthenticated \
+    --update-env-vars FILESTORE_IP_ADDRESS=$$FILESTORE_IP_ADDRESS,FILE_SHARE_NAME=$_SHARE_NAME || \
+    echo 'Failed to build and deploy to Cloud Run' | tee -a /workspace/errors.txt
+
+- id: 'Placeholder for tests'
+  name: 'bash'
```

Comment:
@averikitsch 

This is where I would include the e2e testing - either as multiple steps or as an additional script.

Reply:
Changed - e2e testing will be in `system.test.js`